apps/i18n/common/km_kh.json

```diff
@@ -54,7 +57,7 @@
   "animationCategory_people": "មនុស្ស",
   "animationCategory_retro": "ម៉ត retro",
   "animationCategory_robots": "មនុស្សយន្ត",
-  "animationCategory_space": "លំហ",
+  "animationCategory_space": "space key",
```

Comment:
This regression was added 5 days ago in Crowdin. Not our fault. I let the translator coordinator know.